src/lib/util.typ

```diff
@@ -142,3 +142,12 @@
   set page(header: none, footer: none)
   pagebreak(to: "odd", weak: true)
 }
+
+#let bibstyling(bib-call) = (
+  context {
+    set bibliography(
+      style: "assets/harvard-htl3r.csl",
```

Comment:
Wozu die extra Funktion? Sollte man den Style nicht eher in der Bibliography Page setzen?